ext/exif/extconf.rb

```diff
@@ -5,4 +5,14 @@
 pkg_config('libexif')
 have_library('exif')
 have_header('libexif/exif-data.h')
+
+checking_for 'libexif >= 0.6.23' do
```

Comment:
I thought those tags are added since `0.6.22`?

source: https://github.com/libexif/libexif/releases/tag/libexif-0_6_22-release

Reply:
Good catch, your commit added some tags from 0.6.22 and some from 0.6.23. I added a check for each version, using the `libexif` changelog (ignoring `EXIF_TAG_GAMMA` which is in the changelog but was present for much longer).